src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/ActivityProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Executors;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka
```

Comment:
In my opinion Diagnostic should be a separate namespace that will consists of everything related to telemetry like ActivityProvider/Tags etc.

Reply:
Namespaces help resolve the conflict between two classes (with the same name) in different namespaces.
Also, every class in this extension is in the same namespace "Microsoft.Azure.WebJobs.Extensions.Kafka".
Let's continue following this standard across all classes.